bot/rest/controller/other_controller.py

```diff
@@ -18,3 +20,11 @@ def _routes(self):
         def _active_commanders():
             acs = self._bot.getActiveCommanders()
             return acs if acs is not None else abort(503, "")
+
+        @self.api.route("/registration", methods=["DELETE"])
```

Comment:
You mentioned before that you'd prefer this DELETE-route to have a fully qualified route instead of a payload within the body; I reckon that is a change for another PR, or did you mean to weave it in while you are at it?

Reply:
Since this would be a breaking change I prefer using a separate PR